.eslintrc.json

```diff
@@ -35,20 +27,17 @@
     "import/extensions": "off",
     "react/jsx-uses-vars": "error",
     "react/jsx-uses-react": "error",
+    "react/prop-types": 0,
+    "react/destructuring-assignment": [0, { "extensions": [".jsx"] }],
+    "treatUndefinedAsUnspecified": 0,
     "quotes": [
```

Comment:
Opps, we have some little changes in this file, check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json)

![image](https://user-images.githubusercontent.com/31897434/73417568-ca44bd00-4318-11ea-8a5e-fedabd629b8c.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/